#UAV States
Header header
Geoid geoid
geometry_msgs/Pose pose #NED
geometry_msgs/Twist velocity #body
Accelerations acceleration #body
float64[] rotorspeed #in rad/s